src/App.css

```diff
@@ -1,22 +1,42 @@
 .App {
   text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
   display: flex;
   flex-direction: column;
+  justify-content: space-between;
   align-items: center;
+  height: 100vh;
+}
+
+nav {
+  width: 40%;
+  display: flex;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  align-items: center;
+  margin-bottom: 2%;
+}
+
+nav a {
+  text-decoration: none;
+  color: black;
+  font-size: 1.1rem;
+  padding: 3% 0;
+  margin: 2%;
+  width: 30%;
+  border: 3px solid black;
+}
+
+@media screen and (max-width: 900px) {
+  nav a {
+    width: 50px;
+    height: 50px;
+  }
+
+  .list-link-text {
+    margin-top: 12px;
```

Comment:
Manually specifying a pixel value for margin is a good simple start for centering the text vertically 👍 

But, what if the height of our container changed? Then we would also need to update this pixel value to match.
Fortunately, flexbox gives us a handy way to center content vertically within a container - wiithout needing to specify the pixel height manually!

See this example where I removed the margin, and replaced it with flexbox layout:
![](https://cdn.zappy.app/8c1fef338d724f81434de7ffb40484e6.png)

Feel free to keep this as it is if you want to, it does work fine for now. But, if you want to "flex" your flexbox muscles, this would be a good thing to attempt and learn! 😃

Reply:
And the pun of the PR award goes too... 😆